### PR TITLE
docs: add chocolatey as an install method for windows

### PR DESCRIPTION
**Pull request by VEERT00X**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md].
- [x] The commit message follows the [conventional commits][cc] guidelines.
- [x] Docs have been added/updated (for bug fixes / features).
- [ ] Tests for the changes have been added (for bug fixes / features).

### Description

This pull request aims to address #4654 and focuses on improving the documentation by including information about using Chocolatey as an installation and update method for [Oh My Posh](https://ohmyposh.dev/) on Windows.

image

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: website/docs/installation/windows.mdx [hunk 1]
```diff
@@ -36,6 +36,7 @@ When using oh-my-posh inside the WSL, make sure to follow the [Linux][linux] ins
   values={[
     { label: 'winget', value: 'winget', },
     { label: 'scoop', value: 'scoop', },
+    { label: 'choco', value: 'choco'},
```
Comment (st3phhays) [inline website/docs/installation/windows.mdx]:
```suggestion
    { label: 'chocolatey', value: 'choco'},
```

File: website/docs/installation/windows.mdx [hunk 2]
```diff
@@ -104,6 +116,7 @@ Exclusions should be added with the full path to the executable, you can get it
...
```
Comment (st3phhays) [inline website/docs/installation/windows.mdx]: Same here.

```suggestion
    { label: 'chocolatey', value: 'choco'},
```

**Review by st3phhays — CHANGES_REQUESTED** [review]
Hi there, this is awesome you got a PR in for this. I was the one who raised the issue and am part of the Chocolatey team. 😄 

Would you mind updating the tab label to say "chocolatey" instead of "choco"? I haven't ran this, but I'm assuming this is what shows up for the tab labels at the top. I'm unsure if the value would need to be updated as well, or if it is ok if that still says "choco". The end goal is to have the tab read as "chocolatey".

File: website/docs/installation/windows.mdx (hunk 2 above)
Comment (VEERT00X) [reply website/docs/installation/windows.mdx]: Done

**Review by VEERT00X — COMMENTED** [review]
I have changed the **"label"** _choco_ to _chocolatey_, and refactored the changes to be in the same style as the other labels (label and value are the same). The end result remains the same.

This was tested in a **node** [devcontainer](https://containers.dev/) under **Windows 11 Ubuntu WSL**

---

image
image

**Review by st3phhays — APPROVED** [review]
This LGTM! Thanks for making those changes. 👍

**Comment by JanDeDobbeleer** [comment]
I'm just going to change a few things as this package isn't officially supported by me. And I was always reluctant to add stuff to the docs I can't control 100%.

**Comment by VEERT00X** [comment]
Sure

**Comment by JanDeDobbeleer** [comment]
@all-contributors please add @VEERT00X for doc

This is pure magic 🪄! @holopin-bot @VEERT00X wizard

**Pull request closed without merging**